test/entrypoint.sh

```diff
@@ -19,6 +19,7 @@ wait_tcp_port() {
 	sleep 1
     done
     exec 6>&-
+    echo "Connected to $host:$port!"
```

Comment:
minor nit: can you remove the `!`? I think that will better match the existing `echo`'s.